Sources/SwiftUICharts/LineChart/FullScreenLineChartView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  FullScreenLineChartView.swift
+//
+//
+//  Created by Joshua Holme on 2/9/21.
+//
+
+import SwiftUI
+
+public struct FullScreenLineChartView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
+    @ObservedObject var data:ChartData
```

Comment:
[nit] missing space here

Reply:
This code was copied from elsewhere in the package. Updated in the next commit.